fineract-provider/build.gradle

```diff
@@ -28,6 +28,13 @@ apply plugin: 'com.google.cloud.tools.jib'
 apply plugin: 'org.springframework.boot'
 apply plugin: 'se.thinkcode.cucumber-runner'
 
+
+repositories {
```

Comment:
Please remove this repository!

Reply:
This repository is required for this dependency implementation 'org.pheesdk:PaymentHubSDK:1.0.0'

Reply:
You cannot put your own repository into a community project. If the payment hub sdk is not available on a public repository that is not controlled by your organization, it has absolutely no place in Fineract.

Reply:
Would it be possible to load the Payment Hub SDK dependency at runtime if it's hosted in a private JFrog Artifactory?

Reply:
This is absolutely a no-go. If Payment Hub SDK is needed, let's fix it over there to be published to a publicly accessed repository, for example Maven central.